packages/authentication/src/decorator.ts

```diff
@@ -6,7 +6,8 @@
 import {Reflector, Constructor} from '@loopback/context';
 import {BindingKeys} from './keys';
 
-/* class to hold authentication metadata
+/**
+ * @summary interface to hold authentication metadata
```

Comment:
I don't think we are using `@summary` anywhere else, what is the benefit of using it in this place?